remoting/subsystem/src/main/java/org/jboss/as/remoting/ConnectorChildResource.java

```diff
@@ -63,10 +66,27 @@ static ServiceName getParentServiceName(PathAddress parentAddress) {
 
         protected void populateModel(ModelNode operation, ModelNode model) throws OperationFailedException {
             for (AttributeDefinition ad : attributes) {
+                validateProperty(operation);
                 ad.validateAndSet(operation, model);
             }
         }
 
+        /**
+         * Perform remoting specific validation.
+         * @param operation
+         */
+        private void validateProperty(ModelNode operation) {
+            ModelNode addressModelNode = operation.get(CommonAttributes.ADDRESS);
+            if (!addressModelNode.getType().toString().equals("u")) {
```

Comment:
This condition is hard to understand.

Do you simply mean `if (addressModelNode.isDefined())` ?

Reply:
@jmesnil You are right. I have fixed it.